### ISSUE TITLE
config for all recommended eslint extensions

### ISSUE THREAD

**Issue by vegerot** [vuejs/eslint-plugin-vue#1233]
**Please describe what the rule should do:**

You have a [list](https://eslint.vuejs.org/rules/#extension-rules) of all the eslint extensions you have.  Since most people are probably extending `eslint:recommended` in their config, it would make sense to have a config to enable those same options in `template` tags.  

**What category should the rule belong to?**

- [x ] Enforces code style
- [ x] Warns about a potential error
- [ x] Suggests an alternate way of doing something
- [x] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
          v-if="createDialogType == 'Foo'"
```
```
Expected '===' and instead saw '=='.eslint(vue/eqeqeq)
```

**Additional context**

**Comment by ota-meshi** [comment]
Thank you for this request.

It would be useful for people to be able to provide your suggested shared config, but I think it's difficult to provide it with this plugin due to versioning policies.

If we provided that shared config, we would probably end up changing it frequently. And the change of shared config is a breaking change, so major version release is required.
We have not yet provided all the extended rules that can be applied to the template. Therefore, I will continue to add many extended rules. I don't want to change the sharing config (and major version release) each time we add a rule.

**Comment by vegerot** [comment]
If a third party config were added, would it be possible to reference it from here so people could find it?  Seems like a pretty important thing many people would want

**Comment by ota-meshi** [comment]
If you publish a shareable configuration, you can share it with [awesome-vue].

https://github.com/vuejs/awesome-vue#code-style

[awesome-vue]: https://github.com/vuejs/awesome-vue
